lib/foreman_salt/engine.rb

```diff
@@ -63,4 +63,9 @@ class Engine < ::Rails::Engine
       require 'foreman_salt/extensions'
     end
   end
+
+  # check whether foreman_remote_execution to integrate is available in the system
+  def self.with_remote_execution?
+    RemoteExecutionFeature rescue false
```

Comment:
Wouldn't `defined? RemoteExecutionFeature` do the same thing?

Reply:
adapted from foreman_openscap plugin and similar to foreman_chef.